src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Random random = new Random();
+        ColorSupplier color = new ColorSupplier();
+        int randomNumber = random.nextInt(100);
+        Enum randomColor = color.getRandomColor();
+        Ball ball = new Ball(randomColor, randomNumber);
+
+        return ball.toString();
```

Comment:
If you want to add empty line before return please do the same in all projects methods, or remove empty line here